src/main/java/com/hotels/mutantswarm/MutantSwarmExtension.java

```diff
@@ -144,6 +149,8 @@ public void testFailed(ExtensionContext context, Throwable cause) {
       }
     } else {
       firstTestPassed = false;
+      // If the first test without mutations fails, then we DO throw an exception because it should be successful
+      throw new Exception("Test failed");
```

Comment:
```suggestion
      throw new Exception("Test failed", cause);
```
^ that might give better error message as to why the first test failed?